<ui version="4.0" >
 <author>Bastian Bense</author>
 <class>WidgetSendPrivMsg</class>
 <widget class="QWidget" name="WidgetSendPrivMsg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Private Messages</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QListWidget" name="fUsers" >
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="spacing" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="fSplitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="fMsg" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="tabChangesFocus" >
       <bool>true</bool>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="fInput" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="tabChangesFocus" >
       <bool>true</bool>
      </property>
      <property name="acceptRichText" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fUsers</tabstop>
  <tabstop>fMsg</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
